packages/ui/components/scheduler/components/currentTime/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from "react";
+
```

Comment:
This component doesn't fully work as of right now but I will still explain the what's going on